### PR TITLE
Add `immediateAndChargeFullPrice` proration mode

### PR DESCRIPTION
**Pull request by aboedo**
Adds `immediateAndChargeFullPrice` proration mode

### CODE REVIEW

**Review by vargajacint — APPROVED** [review]
LGTM!

File: __tests__/index.test.js [hunk 1]
```diff
@@ -195,6 +195,38 @@ describe("Purchases", () => {
...
+          title: "title",
+          price: 4.5,
+          priceString: "$4.5",
+          currency_code: "USD",
+          introPrice: null
+        },
+        offeringIdentifier: "offering",
+      },
+      {
+        oldSKU: "viejo",
```
Comment (tonidero) [inline __tests__/index.test.js:220]: I know this comes from the other tests, but I like it :P

**Review by tonidero — APPROVED** [review]

**Comment by vargajacint** [comment]
@aboedo Could you merge and release this, please?

**Comment by aboedo** [comment]
@vegaro could you kick off a release for this?

**Comment by vegaro** [comment]
Doing that now @aboedo

**Comment by vegaro** [comment]
@aboedo released!

**Pull request merged**